extensions/wikia/TriviaQuizzes/package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Why is this file needed?

Reply:
To declare the extension's dependencies (`npm install` or `npm update` will fetch the latest version of content-types-consumption).